docs/developer-docs/integrations/rosetta/index.md

```diff
@@ -34,23 +38,25 @@ To set up a Rosetta node (which connects to a testnet):
         Listening on 0.0.0.0:8080
         Starting Rosetta API server
 
-    By default, the software **does not** connect to the ledger canister running on the Internet Computer blockchain mainnet, but rather it connects to a testnets.
+    By default, the software **does not** connect to the ledger canister running on the Internet Computer blockchain mainnet, it connects to a testnets.
```

Comment:
```suggestion
    By default, the software connects to a testnet. It **does not** connect to the ledger canister running on the Internet Computer blockchain mainnet.
```

Better to avoid double inversion.

Reply:
Fixed.